CesiumJsonReader/src/JsonReaderOptions.cpp

```diff
@@ -25,6 +25,8 @@ class AnyExtensionJsonHandler : public JsonObjectJsonHandler,
         &std::any_cast<CesiumUtility::JsonValue&>(value));
   }
 
+  virtual IJsonHandler& getHandler() override { return *this; }
+
   virtual IJsonHandler* readNull() override {
```

Comment:
Just want to make sure, these other `override` methods are supposed to be here? Is there a reason why `getHandler()` can't return the `JsonObjectJsonHandler` ?

Reply:
> Just want to make sure, these other override methods are supposed to be here?

You're right, they're not needed anymore. Removed.

> Is there a reason why getHandler() can't return the JsonObjectJsonHandler ?

None of the statically-typed extension JsonHandlers derived from JsonObjectJsonHandler.

But it might be sensible for `AnyExtensionJsonHandler` to output an ExtensibleObject with lots of `unknownProperties`, rather than a `JsonValue`, now that we have that unknown properties thing. And then IExtensionJsonHandler::getHandler could return an `ExtensibleObjectJsonHandler`. Probably not really worth the effort to change now, though, and definitely not part of this PR.